### DISCUSSION TITLE
CSS style conflict with reset code

### DISCUSSION THREAD

**Discussion by mahdiar** [vidstack/player#1254] [Q&A]
Hi
I use this reset code for css:
`a,abbr,acronym,address,applet,b,big,blockquote,body,center,cite,code,dd,del,dfn,div,dl,dt,em,fieldset,font,form,h1,h2,h3,h4,h5,h6,html,i,iframe,img,ins,kbd,label,legend,li,object,ol,p,pre,q,s,samp,small,span,strike,strong,sub,sup,tt,u,ul,var{margin:0;padding:0;border:0;outline:0;font-size:100%;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;vertical-align:baseline;background:transparent}`
but span and div styles have problem with player. I think there is a problem in player style.
I attached
![Screenshot from 2024-04-15 11-58-42](https://github.com/vidstack/player/assets/935899/1820f3e4-7a0e-4802-8af7-6513d5406c73)
 the screenshot.

**Comment by mihar-22** [comment]
Thanks for letting us know. This will be fixed in the next release in coming day or two.

**Reply by mahdiar** [reply]
Thanks. There is another problem with volume slider bottom or speed one and the select box (for example for loop) that have not any background color!

**Reply by mahdiar** [reply]
![Screenshot from 2024-05-02 11-12-06](https://github.com/vidstack/player/assets/935899/d59a51bc-2c26-492c-8af9-b4e073086111)
